migrations/20220515_tipo_cuenta_contable.go

```diff
@@ -0,0 +1,71 @@
+package migrations
+
+import (
+	"context"
+
+	"github.com/udistrital/cuentas_contables_crud/models"
+
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/mongo"
```

Comment:
(_suggested change_ menor, mas que todo por "convención")
```suggestion
	migrate "github.com/xakep666/mongo-migrate"
	"go.mongodb.org/mongo-driver/mongo"

	"github.com/udistrital/cuentas_contables_crud/models"
```